org.natron.Natron.json

```diff
@@ -0,0 +1,856 @@
+{
+    "id": "org.natron.Natron",
+    "runtime": "org.freedesktop.Sdk",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "Natron",
+    "rename-desktop-file": "natron.desktop",
+    "rename-appdata-file": "natron.xml",
+    "rename-icon": "natron",
+    "finish-args": [
+            "--share=ipc",
+            "--share=network",
+            "--socket=x11",
+            "--socket=pulseaudio",
+            "--filesystem=host",
+            "--device=dri"
+    ],
+    "modules": [
+            "shared-modules/glew/glew.json",
+    {
```

Comment:
Should be included from shared-modules. (Also, does upstream plan to move to qt5?)